ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml

```diff
@@ -31,21 +35,12 @@ promotion:
     - testopenscap
     name: "4.12"
     namespace: ComplianceAsCode
-releases:
-  initial:
-    integration:
-      name: "4.12"
-      namespace: ocp
-  latest:
-    integration:
-      include_built_images: true
-      name: "4.12"
-      namespace: ocp
 resources:
   '*':
     requests:
       cpu: 200m
       memory: 400Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
```

Comment:
This needs to be adjusted to not rely on `dispatch.sh`, which is stackrox-specific.